### DISCUSSION TITLE
Citations in my own language

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#5127] [Q&A]
I want citations to be in my own language, Spanish.

The problem: In _quarto.yml I have set `lang: es` but the citations are in English, despite before those citations it puts `Por favor, cita este trabajo como:`. For the CSL I use APA7.

**Comment by cderv** [comment]
Can you share an example we can use to reproduce ? I believe we rely on Pandoc citeproc for this, and it should be responsive to the `lang` option. I would like to try with bare Pandoc to see if we get the same results - having your use case would help. thanks !
